client/state/reducer-registry/index.js

```diff
@@ -0,0 +1,152 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { reduce, uniq } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { DESERIALIZE, DESERIALIZE_PART, SERIALIZE } from 'state/action-types';
+import { createReducer, withSchemaValidation } from 'state/utils';
+
+/**
+ * Module variables
+ */
+const registeredReducers = Object.create( null );
+let defaultStoreDispatch;
+
+/**
+ * Pleaseholder reducer keeps undeserialized data in the in-memory state tree
```

Comment:
By `undeserialized` do we mean a plain JS object? Also note the typo in `Pleaseholder`

Reply:
Yes; I should have removed that invented word :-/ The entire comment has been rewritten for clarity.